src/app/modules/main/app_search/module.nim

```diff
@@ -267,3 +267,7 @@ method onSearchMessagesDone*(self: Module, messages: seq[MessageDto]) =
 
 method resultItemClicked*(self: Module, itemId: string) =
   self.controller.resultItemClicked(itemId)
+
+method checkUpdateSearchLocationIfPointToChatWithId*(self: Module, chatId: string) =
```

Comment:
@alexandraB99 I would rename this by just removing `check` at the beginning, since this proc in real updates search location if it points to the passed `chatId`.

Reply:
done!